Packs/GroupIB_ThreatIntelligenceAttribution/Integrations/GroupIBTIA/test_data/example.json

```
@@ -292,62 +292,110 @@
-                    "commonKeywords": 0,
-                    "domain": 0,
-                    "dsn": 0,
-                    "email": 0,
-                    "google": 0,
-                    "ip": 0,
-                    "keyword": 1,
-                    "login": 0,
-                    "metasploit": 0,
-                    "nmap": 0,
-                    "pgp": 0,
-                    "sha": 0,
-                    "slackAPI": 0,
-                    "ssh": 0
+                {
+                    "authorEmail": "some@email.com",
+                    "authorName": "somename"
                 },
-                "name": "conf/nginx/sites-available/whatsinmyyogurt",
-                "repository": "openfoodfacts/openfoodfacts-server",
-                "revisions": [
-                    {
-                        "bind": [{"bindBy": "cert", "data": "cert", "type": "keyword"}],
-                        "data": null,
-                        "file": "https://bt.group-ib.com/api/v2/osi/git_leak/f201c253ac71f7d78db39fa111a2af9d7ee7a3f7/cmV2aXNpb24tZmlsZS01NDA4YWY0MDE2ZTVmZDFjYTZlYWQzNThjYzNiMmI0YjYwNWY1NGY2ODU4Yzc4YmVmMGNlYmUyZGVlMDZmMDhm",
-                        "fileDiff": "https://bt.group-ib.com/api/v2/osi/git_leak/f201c253ac71f7d78db39fa111a2af9d7ee7a3f7/cmV2aXNpb24tZmlsZURpZmYtNTQwOGFmNDAxNmU1ZmQxY2E2ZWFkMzU4Y2MzYjJiNGI2MDVmNTRmNjg1OGM3OGJlZjBjZWJlMmRlZTA2ZjA4Zg==",
-                        "hash": "029eef2ec6e7a16d958f4ae666b7295f427f123b",
-                        "info": {
-                            "authorEmail": "some@gmail.ru",
-                            "authorName": "sadsdsa",
-                            "dateCreated": "2019-06-20T13:13:35+00:00"
+                {
+                    "authorEmail": "some@email.com",
+                    "authorName": "somename"
+                }
+            ],
+            "dataFound": {
+                "password": 8,
+                "apikey": 2,
+                "secret": 1
+            },
+            "dateCreated": "2021-01-23T22:12:58+03:00",
+            "dateDetected": "2021-01-28T22:32:54+03:00",
+            "evaluation": {
+                "admiraltyCode": "A1",
+                "credibility": 50,
+                "reliability": 50,
+                "severity": "orange",
+                "tlp": "amber",
+                "ttl": 30
+            },
+            "favouriteForCompanies": [],
+            "files": [
+                {
+                    "dataFound": [],
+                    "dateCreated": "1970-01-01T03:00:00+03:00",
+                    "dateDetected": "2021-01-28T19:37:08+00:00",
+                    "evaluation": {
+                        "admiraltyCode": "A1",
+                        "credibility": 30,
+                        "reliability": 100,
+                        "severity": "gray",
+                        "tlp": "amber",
+                        "ttl": 30
+                    },
+                    "id": "9ea4c62e9e5da6bc86ecb35e9f4da12fd29cfa5e",
```

Or add it to secrets-ignore
```suggestion
                    "id": "1111111111111111111111111111111111111111",
```